backend/src/main/scala/cromwell/backend/WorkflowBackendActor.scala

```diff
@@ -75,4 +108,19 @@ trait WorkflowBackendActor extends Actor with ActorLogging {
     * Abort a running job.
     */
   def abort(jobKey: BackendJobDescriptorKey): Future[AbortResponse]
+
+  /**
+    * Validate that this WorkflowBackendActor can run all of the calls that it's been assigned
+    */
+  def validate(): Future[InitializationResponse]
+
+  /**
+    * A call which happens before anything else runs
+    */
+  def beforeAll(): Future[Unit]
```

Comment:
It's part of the same InitializationSequence as validate. So we do:
```
final def initSequence() = for {
    _ <- validate()
    _ <- beforeAll()
  } yield InitializationSuccess
```
And if either throws an exception, we get an InitializationFailure instead